src/Symfony/Bridge/Doctrine/Tests/Form/ChoiceList/DoctrineChoiceLoaderTest.php

```diff
@@ -316,7 +316,7 @@ public function testLoadChoicesForValuesLoadsOnlyChoicesIfSingleIntId()
         $this->assertSame(
             [4 => $this->obj3, 7 => $this->obj2],
             $loader->loadChoicesForValues([4 => '3', 7 => '2']
-        ));
+            ));
```

Comment:
That does not look right. 👀

Reply:
To add to my previous comment, I have rejected changes that look very wrong. This one is up for discussion, I suppose. Maybe reverting this is better indeed.

Reply:
This should be:

```php
        $this->assertSame(
            [4 => $this->obj3, 7 => $this->obj2],
            $loader->loadChoicesForValues([4 => '3', 7 => '2'])
        );
```

The closing bracket of the `loadChoicesForValues` call should be on the same line as the opening one. Not sure if it's worth fixing the fixer here though. We could fix that code manually and the fixer should be happy.

Reply:
Oh, you're right, fixed now.